package.json

```diff
@@ -489,6 +489,28 @@
           "group": "2_files",
           "when": "view == remoteExplorer && viewItem != root"
         },
+        {
+          "command": "sftp.upload.folder",
+          "group": "3_trans@1",
+          "alt": "sftp.forceUpload",
+          "when": "sftp.enabled && viewItem != file"
+        },
+        {
+          "command": "sftp.upload.file",
+          "group": "3_trans@1",
+          "when": "view == remoteExplorer && viewItem != root && viewItem == file"
+        },
+        {
+          "command": "sftp.download.folder",
+          "group": "3_trans@2",
+          "alt": "sftp.forceDownload",
+          "when": "sftp.enabled && viewItem != file"
+        },

```

Comment:
@NyaPPuu,
Please report the "_comment" section to the  `Download Folder` command.

Should look like this :
```json
        [...]
        {
          "command": "sftp.download.folder",
          "group": "3_trans@2",
          "alt": "sftp.forceDownload",
          "when": "sftp.enabled && viewItem != file",
          "_comment": "!custom - (using != file so that whole site can be downloaded)"
        }
        [...]
```